src/rdb.h

```diff
@@ -93,6 +93,7 @@
 #define RDB_TYPE_STREAM_LISTPACKS 15
 #define RDB_TYPE_HASH_LISTPACK 16
 #define RDB_TYPE_ZSET_LISTPACK 17
+#define RDB_TYPE_LIST_QUICKLIST_2   18
 /* NOTE: WHEN ADDING NEW RDB TYPE, UPDATE rdbIsObjectType() BELOW */
 
 /* Test if a type is an object type. */
```

Comment:
Forget to change here.
`#define rdbIsObjectType(t) ((t >= 0 && t <= 7) || (t >= 9 && t <= 18))`